### ISSUE TITLE
MegaDrive 6-buttons controller input mapping is passed incorrectly to Mednafen

### ISSUE THREAD

**Issue by SniperJoe** [AmatCoder/mednaffe#134]
When you map up/down/left/right button for MegaDrive 6-buttons controller, Mednaffe passes the mapping with
```bash
-md.input.port1.gamepad.down joystick 0x0003046dc21d40140008000b00000000 abs_7+
```
But the buttons do not work. I have also tried this way of passing input mapping with standalone mednafen:
```bash
mednafen -md.input.port1.gamepad.down "joystick 0x0003046dc21d40140008000b00000000 abs_7+"
```

In order to make it work, the mapping should be passed for 6-buttons controller specifically:
```bash
mednafen -md.input.port1.gamepad6.down "joystick 0x0003046dc21d40140008000b00000000 abs_7+"
```
Everything works fine this way. 

Reproduces on Kubuntu 21.10 with Mednaffe installed via flatpak, with Logitech F310 and DualShock 4 controllers.
I had removed 6-buttons controller input mapping from mednafen.cfg before every test.

**Comment by SniperJoe** [comment]
Have just tested Windows version via Wine, same behavior.

**Comment by haojiezhu** [comment]
I was about to report the same problem. I mapped the 6 buttons to U, I, O, J, K, L key on the keyboard. But somehow X, Y, Z buttons are still mapped to 1, 2, 3 key on the keyboard numpad when I tried "Street Fighters II". To fix this, I had to manually edit the keyboard codes in those lines starting with "md.input.port1.gamepad6" in mednafen.cfg.

**Comment by SniperJoe** [comment]
A little research showed that the problem may be in md.ui and incorrect <property name="command"> values in 6-buttons controller config section.

**Comment by AmatCoder** [comment]
You are right, this was a copy-paste error ;)

Fixed with commit d33001d.

**Comment by AmatCoder** [comment]
Fixed with released version [0.9.2](https://github.com/AmatCoder/mednaffe/releases/latest)
